README.md

```diff
@@ -168,7 +168,7 @@ signature `function(decoded, callback)` where:
         `validateFunc` which can be accessed via `request.plugins['hapi-auth-jwt2'].extraInfo`
 - `validateFunc` - (***required***) the function which is run once the Token has been decoded with
  signature `function(decoded, request, callback)` where:
-    - `decoded` - (***required***) is the ***decoded*** and ***verified*** JWT received from the client in **request.headers.authorization**
+    - `decoded` - (***required***) is the decoded and verified JWT received in the request
```

Comment:
Do you mean the change in styling, or removal of the `request.headers.authorization` reference, or both, make for less detail?